dask_ml/wrappers.py

```diff
@@ -254,7 +255,7 @@ def score(self, X, y, compute=True):
         else:
             return self._postfit_estimator.score(X, y)
 
-    def predict(self, X):
+    def predict(self, X, meta=None):
```

Comment:
This should be a parameter to the `ParallePostFit.__init__` I think not a `predict`.

And then we pass `self.meta` in this method.

Reply:
Just note that if we set `self.meta` then we cant use the same ParallePostFit estimator  on different dask-collections which may have different meta requirements say  `dask-dataframe` , `dask-array` , `dask-cudf dataframe` , `dask-cupy-array` etc.  
   

Just noting that we don't really use `self.meta` in stuff like  `transform` , ` predict_proba` function.  It felt weird having a self.meta that we only use for `predict function`  .   See below: 


### See  def transform(self, X)
https://github.com/dask/dask-ml/blob/f250432bba9046cdca3b464cb6e8eede879d3284/dask_ml/wrappers.py#L206-L213


###  def predict_proba(self, X)
https://github.com/dask/dask-ml/blob/f250432bba9046cdca3b464cb6e8eede879d3284/dask_ml/wrappers.py#L311-L322

I have the following questions: 

1.  Should we pass `self.meta` to all these functions  ?

Reply:
> Just note that if we set self.meta then we cant use the same ParallePostFit estimator on different dask-collections which may have different meta requirements say dask-dataframe , dask-array , dask-cudf dataframe , dask-cupy-array etc.

I'll think about this, but I think that's OK. Users can use `.set_params(meta=my_meta)` to set this.

Reply:
> Should we pass self.meta to all these functions ?

Hmm maybe... But unfortunately, I think `predict` and `predict_proba` might have different `meta` types, since some will return floats and some will return ints, right?

Reply:
> Hmm maybe... But unfortunately, I think predict and predict_proba might have different meta types, since some will return floats and some will return ints, right?

Good Point, They will be different.  We can let `_predict_proba ` as is for now and just over ride `transform` and  `pedict ` and then tackle  `_predict_proba` later ?

Reply:
Sure, sounds good.

Reply:
Done. Addressed all reviews , should be good for another look now.